nixos/modules/services/desktops/flatpak.nix

```diff
@@ -1,7 +1,47 @@
 # flatpak service.
 { config, lib, pkgs, ... }:
+
 let
   cfg = config.services.flatpak;
+
+  flatpakCommand = "${cfg.package}/bin/flatpak";
```

Comment:
```suggestion
  flatpakCommand = lib.getExe cfg.package;
```
does that work here?

Reply:
Although it's working but changing to `lib.getExe cfg.package` gives evaluation error
```
trace: evaluation warning: getExe: Package "flatpak-1.14.10" does not have the meta.mainProgram attribute.
```

Reply:
Thus, I think I'll change the `flatpakCommand` back to `"${cfg.package}/bin/flatpak"`